roles/grafana_agent/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+# defaults file for agent
+
+# Mimir URL and creds
+agent_mimir_url: "http://sepia-grafana.front.sepia.ceph.com:9009/api/v1/push"
+agent_mimir_username: "admin"
+
+# Enable metrics globally
+agent_metrics: true
+
+# Labels used for external labels
+nodetype: "unknown_nodetype"
```

Comment:
Why is this a variable? Can we hard-code it into `grafana-agent.yaml.j2` for simplicity?

Reply:
Those labels are used in Prometheus to distinguish between the different machines groups